src/uniprotkb/config/UniProtKBColumnConfiguration.tsx

```diff
@@ -609,6 +607,22 @@ UniProtKBColumnConfiguration.set(UniProtKBColumn.ccSequenceCaution, {
   },
 });
 
+UniProtKBColumnConfiguration.set(UniProtKBColumn.featureCount, {
+  label: 'Features',
+  render: (data) => {
+    const counts = data?.extraAttributes?.countByFeatureType;
+    return (
+      counts && (
+        <ExpandableList displayNumberOfHiddenItems>
```

Comment:
How about:
```suggestion
        <ExpandableList displayNumberOfHiddenItems descriptionString="features">
```
Not strongly attached though, but we do do that in other places where we know what the content of the list is